desc/compute/_deprecated.py

```diff
@@ -0,0 +1,382 @@
+    """Fast ion confinement proxy as defined by Nemov et al.
+
+    Poloidal motion of trapped particle orbits in real-space coordinates.
+    V. V. Nemov, S. V. Kasilov, W. Kernbichler, G. O. Leitold.
+    Phys. Plasmas 1 May 2008; 15 (5): 052501.
+    https://doi.org/10.1063/1.2912456.
+    Equation 61.
+
+    A 3D stellarator magnetic field admits ripple wells that lead to enhanced
+    radial drift of trapped particles. The energetic particle confinement
+    metric γ_c quantifies whether the contours of the second adiabatic invariant
+    close on the flux surfaces. In the limit where the poloidal drift velocity
+    majorizes the radial drift velocity, the contours lie parallel to flux
+    surfaces. The optimization metric Γ_c averages γ_c² over the distribution
+    of trapped particles on each flux surface.
+
+    The radial electric field has a negligible effect, since fast particles
+    have high energy with collisionless orbits, so it is assumed to be zero.
+    """
+    # noqa: unused dependency
+    num_pitch = kwargs.get("num_pitch", 64)
+    num_well = kwargs.get("num_well", None)
+    quad = (
+        kwargs["quad"]
+        if "quad" in kwargs
+        else get_quadrature(
+            leggauss(kwargs.get("num_quad", 32)),
+            (automorphism_sin, grad_automorphism_sin),
+        )
+    )
+
+    def Gamma_c(data):
+        """∫ dλ ∑ⱼ [v τ γ_c²]ⱼ π²/4."""
+        bounce = Bounce1D(grid, data, quad, is_reshaped=True)
+        points = bounce.points(data["pitch_inv"], num_well=num_well)
+        v_tau, drift1, drift2 = bounce.integrate(
+            [_v_tau, _drift1, _drift2],
+            data["pitch_inv"],
+            data,
+            ["|grad(psi)|*kappa_g", "|B|_r|v,p", "K"],
+            points,
+        )
+        # This is γ_c π/2.
+        gamma_c = jnp.arctan(
+            safediv(
+                drift1,
+                drift2
+                * bounce.interp_to_argmin(data["|grad(rho)|*|e_alpha|r,p|"], points),
+            )
+        )
+        return jnp.sum(
+            jnp.sum(v_tau * gamma_c**2, axis=-1)
+            * data["pitch_inv weight"]
+            / data["pitch_inv"] ** 2,
+            axis=-1,
+        )
+
+    grid = transforms["grid"].source_grid
+    data["deprecated(Gamma_c)"] = _compute(
+        Gamma_c,
+        fun_data={
+            "|grad(psi)|*kappa_g": data["|grad(psi)|"] * data["kappa_g"],
+            "|grad(rho)|*|e_alpha|r,p|": data["|grad(rho)|"] * data["|e_alpha|r,p|"],
+            "|B|_r|v,p": data["|B|_r|v,p"],
+            "K": data["iota_r"]
+            * dot(cross(data["grad(psi)"], data["b"]), data["grad(phi)"])
+            - (
+                2 * data["|B|_r|v,p"]
+                - data["|B|"] * data["B^phi_r|v,p"] / data["B^phi"]
+            ),
+        },
+        data=data,
+        grid=grid,
+        num_pitch=num_pitch,
+        simp=False,
+    ) / (data["fieldline length"] * 2**1.5 * jnp.pi)
+    return data
+
+
+def _gbdrift(data, B, pitch):
```

Comment:
We are calculating the numerator of the bounce integral.
Can we try a different name? Not important.

Reply:
done in different PR